CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [qdrant-1.15.1](https://github.com/qdrant/qdrant-helm/tree/qdrant-1.15.1) (2025-07-21)
```

Comment:
Please do not add a changelog the PR. If merged, we will automatically create the changelog with all PRs included in the next release.

Reply:
Fixed